platform/xbox/functions/_PDCLIB/_PDCLIB_w32errno.c

```diff
@@ -0,0 +1,54 @@
+/* _PDCLIB_w32errno( void )
+
+   This file is part of the Public Domain C Library (PDCLib).
+   Permission is granted to use, modify, and / or redistribute at will.
+*/
+
+#ifndef REGTEST
+#include <errno.h>
+#include <wchar.h> // Watcom bug: winnt.h assumes string.h defines wchar_t
+#include <windows.h>
+
+void _PDCLIB_w32errno(void);

```

Comment:
This prototype seems fairly pointless?

Reply:
Probably, but as I said above, I didn't touch the file contents.